src/representation_theory/Rep.lean

```diff
@@ -71,6 +72,50 @@ by apply_instance
 noncomputable example : preserves_colimits (forget₂ (Rep k G) (Module.{u} k)) :=
 by apply_instance
 
+section linearisation
```

Comment:
Mathlib seems to mostly use american spelling. I don't think there is a strict policy, but it might be the most predictable...